### PR TITLE
Remove bootstrap css from jobs page

### PR DESCRIPTION
**Pull request by cp-shruti-s**
Remove bootsrap css from jobs page 

- [x] Landing section
- [x] Virturs section
- [x] Life at canopas section
- [x] Career section
- [x] Perks and benefit section
- [x] Why canopas section
- [x] FAQ section

### CODE REVIEW

File: vue-frontend/src/components/jobs/LifeAtCanopas.vue [hunk 1]
```diff
@@ -1,17 +1,19 @@
 <template>
-  <div class="pb-32">
-    <div class="life-container normal-text text-center">
+  <div class="life-at-canopas">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/LifeAtCanopas.vue:2]: Will this class be used? If yes, let's add tailwind CSS for the same

**Review by cp-sumi-k — APPROVED** [review]

File: vue-frontend/src/components/jobs/LifeAtCanopas.vue (hunk 1 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/jobs/LifeAtCanopas.vue:2]: No this class is not used any were. this class just for identification this is life at canopas section

Comment (cp-sumi-k) [reply vue-frontend/src/components/jobs/LifeAtCanopas.vue:2]: Let's remove it, as we are not adding these type of class anywhere

Comment (cp-shruti-s) [reply vue-frontend/src/components/jobs/LifeAtCanopas.vue:2]: okay

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**